api/xml/libero/core/inline.rng

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<grammar xmlns="http://relaxng.org/ns/structure/1.0"
+    datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
+
+    <define name="libero.text.i">
+        <element name="i">
+            <ref name="libero.text.model"/>
+        </element>
+    </define>
+
+    <define name="libero.text.class">
```

Comment:
is `class` common terminology from `relaxng` schemas?

Reply:
Not sure. I borrow these ideas from the XHTML 2.0 (ahem) schema - https://www.w3.org/TR/xhtml2/relax_module_defs.html.